com.gitlab.adnan338.Nixwriter.yaml

```diff
@@ -0,0 +1,38 @@
+id: com.gitlab.adnan338.Nixwriter
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: nixwriter
+finish-args:
+  - --socket=fallback-x11
```

Comment:
I think you forgot to add wayland here as well

Reply:
Added. The documentation says "--socket=fallback-x11:  show windows using X11, if Wayland is not available, overrides x11 socket permission". It should mention that adding `--socket=wayland` is still required.

Reply:
Please open an issue against https://github.com/flatpak/flatpak-docs/ for that, or even better, add the missing bit to make the sentence clearer :)